Source/Core/DolphinQt/GCMemcardManager.cpp

```diff
@@ -15,25 +20,40 @@
 #include <QImage>
 #include <QLabel>
 #include <QLineEdit>
+#include <QMenu>
 #include <QPixmap>
 #include <QPushButton>
 #include <QString>
 #include <QStringList>
 #include <QTableWidget>
 #include <QTimer>
+#include <QToolButton>
 
+#include "Common/CommonPaths.h"
 #include "Common/Config/Config.h"
 #include "Common/FileUtil.h"
 #include "Common/MsgHandler.h"
 #include "Common/StringUtil.h"
 
 #include "Core/Config/MainSettings.h"
 #include "Core/HW/GCMemcard/GCMemcard.h"
+#include "Core/HW/GCMemcard/GCMemcardUtils.h"
 
 #include "DolphinQt/GCMemcardCreateNewDialog.h"
 #include "DolphinQt/QtUtils/ModalMessageBox.h"
 
-constexpr float ROW_HEIGHT = 28;
+constexpr int ROW_HEIGHT = 36;
+constexpr int COLUMN_WIDTH_FILENAME = 100;
+constexpr int COLUMN_WIDTH_BANNER = 96 + 6;
```

Comment:
Any reason why this isn't just 102?

Reply:
96 is the width of the banner and then the 6 is just to give it a bit of space in the table, IIRC. Probably not a bad idea to give that a name though.